server/src/main/java/org/elasticsearch/search/retriever/RankDocsRetrieverBuilder.java

```diff
@@ -32,14 +32,16 @@ public class RankDocsRetrieverBuilder extends RetrieverBuilder {
     final int rankWindowSize;
     final List<RetrieverBuilder> sources;
     final Supplier<RankDoc[]> rankDocs;
+    final Float minScore;
```

Comment:
Can we please note add a member for this, and use/update the `minScore` that is defined in the parent `RetrieverBuilder`?